src/Solver.jl

```diff
@@ -42,6 +42,12 @@ function setup!(solver::Solver, matrix::AMGXMatrix)
     return solver
 end
 
+function resetup!(solver::Solver, matrix::AMGXMatrix)
```

Comment:
How are people supposed to know about this function and know what it does? It isn't in the README I think?